test/NomoPlayersDropMechanic.test.ts

```diff
@@ -390,11 +390,115 @@ describe("NomoPlayersDropMechanic tests", function () {
     });
 
     it("must fail to set whitelisted if whitelisted array is over 100", async function () {
-      const exceedingLimitNumberOfAddr = WHITELISTED_TEST_ADDRESSES.slice(0, 101);
+      const exceedingLimitNumberOfAddr = TEST_ADDRESSES.slice(0, 101);
       await expect(nomoPlayersDropMechanicContract.connect(deployer).setWhitelisted(exceedingLimitNumberOfAddr)).to.be.revertedWith("Beneficiaries array length must be in the bounds of 1 and 100");
     });
   });
 
+  context("for privileged", () => {
```

Comment:
should have a test covering a scenario, that the claimed tokens for each privileged account is incremented